content/docs/command-reference/add.md

```diff
@@ -37,29 +37,29 @@ Under the hood, a few actions are taken for each file (or directory) in
    for more details.)
 3. Attempt to replace the file with a link to the cached data (more details
    further down).
-4. Create a corresponding DVC-file to store the file (as an
+4. Create a corresponding `.dvc` file to store the file (as an
    <abbr>output</abbr>), using its path and hash to identify the cached data.
-   Unless the `-f` option is used, the DVC-file name generated by default is
+   Unless the `-f` option is used, the `.dvc` file name generated by default is
    `<file>.dvc`, where `<file>` is the file name of the first target.
```

Comment:
In this case it's obvious what kind of file we're talking about, from the context.
```suggestion
   Unless the `-f` option is used, the file name generated by default is
   `<file>.dvc`, where `<file>` is the file name of the first target.
```

Reply:
Actually no, never mind.